CONTRIBUTING.md

```diff
@@ -32,15 +32,63 @@ Follow [@zitadel](https://twitter.com/zitadel) on twitter
 
 ## How to contribute
 
-We strongly recommend to talk to us before you start contributing to streamline our and your work.
+We strongly recommend to [talk to us](https://zitadel.com/contact) before you start contributing to streamline our and your work.
 
 We accept contributions through pull requests. You need a github account for that. If you are unfamiliar with git have a look at Github's documentation on [creating forks](https://help.github.com/articles/fork-a-repo) and [creating pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). Please draft the pull request as soon as possible. Go through the following checklist before you submit the final pull request:
 
-1. Create a feature branch from the `main`-branch
-1. Make your changes on the new branch
-1. [Merge](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) the latest commit of the `main`-branch
-1. Use [Semantic Release commit messages](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type) to simplify creation of release notes. In the title of the pull request correct tagging is required and will be requested by the reviewers.
-1. Request a [review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from one of the authors. The reviewers will provide you feedback and approve your changes as soon as they are satisfied.
+### Submit a Pull Request (PR)
+
+> :warning: Currently main development is done on branch `v2-alpha`. Make sure you're merging into the correct branch.
+
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the zitadel/zitadel repository on Github
+2. On your fork, commit your changes to a new branch
+
+`git checkout -b my-fix-branch main`
+
+3. Make your changes following the [guidelines](#contribute) in this guide. Make sure that all tests pass.
+
+4. Commit the changes on the new branch
+
+`git commit --all`
+
+5. [Merge](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) the latest commit of the `main`-branch
```

Comment:
```suggestion
5. [Merge](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) the latest commit of the `v2-alpha`-branch
```

Reply:
I've added the warning on top. Do you think that's not enough?

Reply:
I am torn with this one, because most of the times user simply copie stuff 😁 

But its not a deal breaker to merge this